include/solvers/adaptive_time_solver.h

```diff
@@ -22,7 +22,7 @@
 
 // Local includes
 #include "libmesh/system_norm.h"
-#include "libmesh/unsteady_solver.h"
```

Comment:
You know?  I don't understand that either.  I probably mistakenly added some unfortunate redundancy when factoring out TwostepTimeSolver, and the correct solution is probably to just hand off to the core_time_solver.

On the other hand, "I don't understand that either" means Chesterton's fence applies here.  I've got a bunch of private apps that use this class, but we *don't* yet have any regression test coverage of it.  So let's leave it the way you've got it here, and we can reconsider refactoring it later.